app/models/concerns/client_search.rb

```diff
@@ -51,6 +51,14 @@ def self.text_searcher(text, sorted:, resolve_for_join_query: false)
           )
           where = conditions.reduce(:or)
         end
+      elsif /household:[[[:alnum:]]-]+/.match(text)
+        household_id = text.gsub('household:', '')
+        client_ids = GrdaWarehouse::Hud::Enrollment.where(HouseholdID: household_id).joins(:client).pluck(sa[:id])
```

Comment:
Another options would be to just search Personal IDs, e.g. `GrdaWarehouse::Hud::Enrollment.where(HouseholdID: household_id).pluck(:PersonalID); sa[:PersonalID].in(personal_ids)` . I chose not to do that because it could lead to errant clients showing up from other data sources when used in the warehouse.

Reply:
I think this is great.  I like that we're starting to support faceted searching.  I wonder if the regex should allow whitespace between `household:` and the values, to support `household: 123`.  Even an optional space would probably be welcome.  That said, if the front-end is mashing these together from a custom search field specifically for household, then not having the space is totally good.

Reply:
👍  Added support for the space, why not